### ISSUE TITLE
Does not handle Tamil SIGN VIRAMA and VOWEL SIGN I

### ISSUE THREAD

**Issue by Haroenv** [dotcypress/runes#2]
```
   0BA8 ├─ ந		├─ TAMIL LETTER NA
   0B95 ├─ க		├─ TAMIL LETTER KA
   0BB0 ├─ ர		├─ TAMIL LETTER RA
   ---- ├┬ த்		├┬ Composition: TAMIL CONSONANT T
   0BA4 │├─ த		│├─ TAMIL LETTER TA
   0BCD │└─ ◌்		│└─ TAMIL SIGN VIRAMA
   ---- ├┬ தி		├┬ Composition: TAMIL SYLLABLE TI
   0BA4 │├─ த		│├─ TAMIL LETTER TA
   0BBF │└─ ி		│└─ TAMIL VOWEL SIGN I
   ---- ├┬ ல்		├┬ Composition: TAMIL CONSONANT L
   0BB2 │├─ ல		│├─ TAMIL LETTER LA
   0BCD │└─ ◌்		│└─ TAMIL SIGN VIRAMA
```

```js  
const tamil = 'நகரத்தில்';

expect(runes(tamil)).toEqual(['ந', 'க', 'ர', 'த்', 'தி', 'ல்']);
```

```diff
Expected value to equal:
["ந", "க", "ர", "த்", "தி", "ல்"]
Received:
["ந", "க", "ர", "த", "்", "த", "ி", "ல", "்"]

Difference:

- Expected
+ Received

Array [
 "ந",
 "க",
 "ர",
-  "த்",
-  "தி",
-  "ல்",
+  "த",
+  "்",
+  "த",
+  "ி",
+  "ல",
+  "்",
]
```

**Comment by Haroenv** [comment]
Similar for Thai: 

```js
  // ---- ├┬ ม้		├┬ Composition
  // 0E21 │├─ ม		│├─ THAI CHARACTER MO MA
  // 0E49 │└─ ◌้		│└─ THAI CHARACTER MAI THO
  // 0E32 ├─ า		├─ THAI CHARACTER SARA AA
  // 0E25 ├─ ล		├─ THAI CHARACTER LO LING
  // 0E32 ├─ า		├─ THAI CHARACTER SARA AA
  // 0E22 ├─ ย		├─ THAI CHARACTER YO YAK
  // 0E2B ├─ ห		├─ THAI CHARACTER HO HIP
  // 0E01 ├─ ก		├─ THAI CHARACTER KO KAI
  // ---- ├┬ ตั		├┬ Composition
  // 0E15 │├─ ต		│├─ THAI CHARACTER TO TAO
  // 0E31 │└─ ◌ั		│└─ THAI CHARACTER MAI HAN-AKAT
  // 0E27 ├─ ว		├─ THAI CHARACTER WO WAEN
  const thai = 'ม้าลายหกตัว';
  expect(runes(thai)).toEqual(['ม้', 'า', 'ล', 'า', 'ย', 'ห', 'ก', 'ตั']);
```
```diff
Expected value to equal:
  ["ม้", "า", "ล", "า", "ย", "ห", "ก", "ตั"]
Received:
  ["ม", "้", "า", "ล", "า", "ย", "ห", "ก", "ต", "ั", "ว"]

Difference:

- Expected
+ Received

Array [
-  "ม้",
+  "ม",
+  "้",
   "า",
   "ล",
   "า",
   "ย",
   "ห",
   "ก",
-  "ตั",
+  "ต",
+  "ั",
+  "ว",
 ]
```

**Comment by Haroenv** [comment]
relevant lecture: http://unicode.org/reports/tr29/

**Comment by dotcypress** [comment]
Awesome!
Thank you

**Comment by dotcypress** [comment]
Fix landed in `0.4.2`
